admin/css/admin.css

```diff
@@ -427,6 +427,14 @@ body.post-type-volunteer_opp .wp-pointer-arrow {
 }
 
 /**
+* Widget styling
+*/
+.wi-widget-block-label:after {
```

Comment:
@jg314 @caseynix 
Yes, the reason I used a pseudo-element with the after selector is that if I simply set .wi-widget-block-label to `display: block;` then the labels next to the checkboxes will fall under the checkboxes instead of being next to them. I've attached a screenshot to show this at the end of this comment.

An alternative method that I could use is to wrap each input and related label in its own <p> tag OR use <br> tags. I would prefer not to use <br> tags in this instance but would be open to converting the markup to use the <p> tag alternative if you would prefer?

<img width="342" alt="screen shot 2016-05-03 at 8 45 49 pm" src="https://cloud.githubusercontent.com/assets/3980089/15003294/1a78e7e4-1170-11e6-948b-9132d7e0ce95.png">